tfjs-backend-webgpu/src/binary_op_util.ts

```diff
@@ -113,8 +114,11 @@ const LESS_VEC4 = 'return vec4<f32>(a < b);';
 const LESS_EQUAL = 'return f32(a <= b);';
 const LESS_EQUAL_VEC4 = 'return vec4<f32>(a <= b);';
 const LOGICAL_AND = 'return f32(f32(a) >= 1.0 && f32(b) >= 1.0);';
-const LOGICAL_AND_VEC4 = `return (vec4<f32>(a >= vec4<f32>(1.0)) *
-  vec4<f32>(b >= vec4<f32>(1.0)));`;
+const LOGICAL_AND_VEC4 = `return (vec4<f32>(vec4<f32>(a) >= vec4<f32>(1.0)) *
```

Comment:
nit: I think `a` and `b` are already `vec4<f32>` type. So you can simply change `vec4<f32>(a)` to `a`.
And please also change`LOGICAL_AND` and belows.